egs2/TEMPLATE/svs1/README.md

```diff
@@ -0,0 +1,540 @@
+
+### Evaluation
+
+We provide four objective evaluation metrics:
+
+- Mel-cepstral distortion (MCD)
+- Logarithmic rooted mean square error of the fundamental frequency (F![1](http://latex.codecogs.com/svg.latex?_0)RMSE)
+- Semitone accuracy (Semitone ACC)
+- Voiced / unvoiced error rate (VUV_E)
+
+For MCD, we apply dynamic time-warping (DTW) to match the length difference between ground-truth singing and generated singing.
+
+Here we show the example command to calculate objective metrics:
+
+```sh
+cd egs/<recipe_name>/svs1
+. ./path.sh
+# Evaluate MCD
+./pyscripts/utils/evaluate_mcd.py \
+    exp/<model_dir_name>/<decode_dir_name>/eval/wav/gen_wavdir_or_wavscp.scp \
+    dump/raw/eval/gt_wavdir_or_wavscp.scp
+```
+```sh
+cd egs/<recipe_name>/svs1
+. ./path.sh
+# Evaluate log-F0 RMSE & Semitone ACC & VUV Error Rate
+./pyscripts/utils/evaluate_f0.py \
+    exp/<model_dir_name>/<decode_dir_name>/eval/wav/gen_wavdir_or_wavscp.scp \
+    dump/raw/eval/gen_wavdir_or_wavscp.scp
+```
+While these objective metrics can estimate the quality of synthesized singing, it is still difficult to fully determine human perceptual quality from these values, especially with high-fidelity generated singing.
+Therefore, we recommend performing the subjective evaluation (eg. MOS) if you want to check perceptual quality in detail.
+
+You can refer [this page](https://github.com/kan-bayashi/webMUSHRA/blob/master/HOW_TO_SETUP.md) to launch web-based subjective evaluation system with [webMUSHRA](https://github.com/audiolabs/webMUSHRA).
+
+## About data directory
+
+Each directory of training set, development set, and evaluation set, has same directory structure. See also https://github.com/espnet/espnet/tree/master/egs2/TEMPLATE#about-kaldi-style-data-directory about Kaldi data structure. 
+We recommend you running `ofuton_p_utagoe_db` recipe and checking the contents of `data/` by yourself.
+
+```bash
+cd egs/ofuton_p_utagoe_db/svs1
+./run.sh
+```
+
+- Directory structure
+    ```
+    data/
+    ├── tr_no_dev/     # Training set directory
+    │   ├── text       # The transcription
+    │   ├── label      # Specifying start and end time of the transcription 
+    │   ├── midi.scp   # MIDI file path
+    │   ├── wav.scp    # Wave file path
+    │   ├── utt2spk    # A file mapping utterance-id to speaker-id
+    │   ├── spk2utt    # A file mapping speaker-id to utterance-id
+    │   ├── segments   # [Option] Specifying start and end time of each utterance
+    │   └── (utt2lang) # A file mapping utterance-id to language type (only for multilingual)
+    ├── dev/
+    │   ...
+    ├── test/
+    │   ...
+    └── token_list/   # token list directory
+        ...
+    ```
+
+ - `text` format
+    ```
+    uttidA <transcription>
+    uttidB <transcription>
+    ...
+    ```
+    
+- `label` format
+    ```
+    uttidA (startA1, endA1, phA1) (startA2, endA2, phA1) ...
+    uttidB (startB1, endB1, phB1) (startB2, endB2, phB2) ...
+    ...
+    ```
+    
+- `midi.scp` format
```

Comment:
Can you double-check with @A-Quarter-Mile whether this is still hold?

Reply:
We can add a note here. It's prepared for recipes with only MIDI for pitch information.

Reply:
I can't find where to add a link in espnet/README.md.

Reply:
That's no problem. I can add that later